extensions/amp-story/1.0/amp-story-open-page-attachment.js

```diff
@@ -108,6 +127,75 @@ const renderDefaultPageAttachmentUI = (pageEl, attachmentEl) => {
   return openAttachmentEl;
 };
 
+/**
+ * Renders inline page attachment UI.
+ * @param {!Window} win
+ * @param {!Element} pageEl
+ * @param {!Element} attachmentEl
+ * @return {!Element}
+ */
+ const renderOutlinkPageAttachmentUI = (win, pageEl, attachmentEl, attachmentHref) => {
+  const openAttachmentEl = buildOpenOutlinkAttachmentElement(pageEl);
+
+  // Copy href to the element so it can be previewed on hover and long press.
+  if (attachmentHref) {
+    openAttachmentEl.setAttribute('href', attachmentHref);
+  }
+  
+  openAttachmentEl.addEventListener('click', () =>
+    openAttachmentEl.openAttachment()
+  );
+
+  // Setting theme  
+  const theme = attachmentEl.getAttribute('theme');
+  if (theme && AttachmentTheme.DARK === theme.toLowerCase()) {
+    openAttachmentEl.setAttribute('theme', AttachmentTheme.DARK);
+  }
+
+  // Appending text & aria-label.
+  const openLabelAttr = attachmentEl.getAttribute('data-cta-text');
+  const openLabel =
+    (openLabelAttr && openLabelAttr.trim()) ||
+    getLocalizationService(pageEl).getLocalizedString(
+      LocalizedStringId.AMP_STORY_PAGE_ATTACHMENT_OPEN_LABEL
+    );
+
+  const ctaLabelEl = openAttachmentEl.querySelector(
+    '.i-amphtml-story-outlink-page-attachment-label'
+  );
+  ctaLabelEl.innerHTML = openLabel;
+  openAttachmentEl.setAttribute('aria-label', openLabel);
+
+  // Adding image.
+  const openImgAttr = attachmentEl.getAttribute('cta-image');
+  const chipEl = openAttachmentEl.querySelector(
```

Comment:
Check the method `htmlRefs` which allows you to do all this without querySelectors, and directly extract a lot of elements from a template on one line